router_categories_test.go

```diff
@@ -187,7 +187,7 @@ func TestCategoriesDeleteAdmin(t *testing.T) {
 		assert.NoError(t, json.Unmarshal(*bslice, &result))
 		assert.True(t, ok)
 
-		postCount, _, ok := getCategoriesWithCount(t)
+		postCount, _, _ := getCategoriesWithCount(t)
```

Comment:
Why not check for `ok` as well?

```suggestion
		postCount, _, ok := getCategoriesWithCount(t)
		assert.True(t, ok)
```

Reply:
Sure!